lib/applicants/resolvers.ts

```diff
@@ -458,6 +459,16 @@ export const verifyIdentity: Resolver<MutationVerifyIdentityArgs, VerifyIdentity
     };
   }
 
+  // Note: 30 days = 30 * 24 * 60 * 60 * 1000 milliseconds
+  //6 months past expiry date
+  if (moment(mostRecentPermit.expiryDate).add(6, 'M') >= moment()) {
```

Comment:
`momentRecentPermit.expiryDate` is inputed as a UTC Datetime object. (i.e. YYYY-MM-DD) so this code will convert it to our local time zone which we do not want I believe. I think the proper code should be `moment.utc(mostRecentPermit.expiryDate).add(6, 'M') >= mpment())` but this needs to be verified via testing.